packages/viewer/src/types/animation.ts

```diff
@@ -0,0 +1,17 @@
+export interface Animation {
+  milliseconds?: number;
+  easing?:
+    | 'linear'
+    | 'ease-out-cubic'
+    | 'ease-out-quad'
+    | 'ease-out-quart'
+    | 'ease-out-sine'
+    | 'ease-out-expo';
```

Comment:
IMO `milliseconds` should be a required fields, and could be convinced that `easing` is as well.